internal-services/catalog/pulp-push-disk-images-task.yaml

```diff
@@ -31,9 +31,29 @@ spec:
     - name: udcacheSecret
       type: string
       description: Env specific secret containing the udcache credentials
+    - name: cgwHostname
+      type: string
+      description: Env specific hostname for content gateway
+    - name: cgwSecret
+      type: string
+      description: Env specific secret containing the content gateway credentials
+    - name: productName
+      type: string
+      description: product name
+    - name: productCode
+      type: string
+      description: product code
+    - name: productVersionName
+      type: string
+      description: product version name
+    - name: component_json
+      type: string
+      description: String containing a JSON representation of the components listed in the data.mapping in RPA
   results:
     - name: result
       description: Success if the task succeeds, the error otherwise
+    - name: download-dir
```

Comment:
This doesn't seem used, and I don't really see what the value of it could be. Why would we want to say a dir name in a pod as a result of the InternalRequest pipelineRun and pass that up to the managed pipelineRun which isn't even on the same cluster? I am assuming that is why we'd want the result because so far the only use of IR plr results is to return the data to the managed plr

Reply:
removed